.github/workflows/publish.yml

```diff
@@ -12,12 +12,17 @@ jobs:
       - uses: linz/action-typescript@dee99184c4305aea6c380a52db9b2d7abaaa3e78 # v3
 
       # FIXME: catalog.json is not pushed to the repository (temporary solution)
-      - name: Create Catalog
+      - name: Create STAC Catalog
         uses: docker://ghcr.io/linz/argo-tasks:v2
         with:
           args: stac-catalog --output stac/catalog.json --template template/catalog.json /github/workspace/stac/
 
-      - name: Validate STAC
+      - name: Validate STAC Catalog
+        uses: docker://ghcr.io/linz/argo-tasks:v2
+        with:
+          args: stac-validate /github/workspace/stac/catalog.json
+
+      - name: Validate STAC Collections
```

Comment:
If not to the question above, why not using the same pattern using the docker image with args? (see above)

Reply:
Sorry my message was confusing I meant using this pattern:
```
name: Create STAC Catalog
        uses: docker://ghcr.io/linz/argo-tasks:v2
        with:
          args:
```
Which is "fixing" the existing `Validate STAC Collections` step.